src/providers/aws.rs

```diff
@@ -739,6 +739,7 @@ mod test {
     use rusoto_ec2::Ec2;
 
     #[test]
+    #[ignore]
```

Comment:
It'd be awesome to configure Travis so that it had a local SSH endpoint we could connect to (perhaps a container) with a locally generated keypair. That's probably outside the scope of this PR though.